Formula/libvisual-0.4.rb

```diff
@@ -0,0 +1,23 @@
+class Libvisual04 < Formula
```

Comment:
```suggestion
class Libvisual < Formula
```
We don't need this formula to be versioned.

Reply:
We're a rolling release package manager, so the names of formulae always refer to the latest stable (i.e. general use) version. If we need a [versioned formula](https://docs.brew.sh/Versions), we can consider adding one later, but we should cross that bridge when we get there.